iotdb-core/datanode/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -1271,7 +1278,7 @@ private void loadWALHotModifiedProps(Properties properties) {
         Long.parseLong(
             properties.getProperty(
                 "iot_consensus_throttle_threshold_in_byte",
-                Long.toString(conf.getThrottleThreshold())));
+                Long.toString(getThrottleThresholdWithDirs())));
```

Comment:
if iot_consensus_throttle_threshold_in_byte is set, we do not need to calculte getThrottleThresholdWithDirs